scripts/protocgen.sh

```diff
@@ -2,42 +2,25 @@
 
 set -eo pipefail
 
-protoc_gen_gocosmos() {
-  if ! grep "github.com/gogo/protobuf => github.com/regen-network/protobuf" go.mod &>/dev/null ; then
-    echo -e "\tPlease run this command from somewhere inside the sommelier folder."
-    return 1
-  fi
+# get protoc executions
+go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
 
-  go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
-}
-
-protoc_gen_doc() {
-  go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc 2>/dev/null
-} 
-
-protoc_gen_gocosmos
-protoc_gen_doc
+# get cosmos sdk
+go get github.com/cosmos/cosmos-sdk 2>/dev/null
```

Comment:
What is the purpose of this? Without version?

Reply:
No purpose, thank you for pointing this out